<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WoW Console Runner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphics.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="widget_main">
   <layout class="QHBoxLayout" name="layout_main">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>24</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>24</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="layout_polybius">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="console_polybius">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::WidgetWidth</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>     ___      _       _     _
    / _ \___ | |_   _| |__ (_)_   _ ___
   / /_)/ _ \| | | | | '_ \| | | | / __|
  / ___/ (_) | | |_| | |_) | | |_| \__ \
  \/    \___/|_|\__, |_.__/|_|\__,_|___/
                |___/

</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_exe_polybius">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_exe_polybius">
          <property name="text">
           <string>Executable Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_exe_polybius">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_exe_polybius"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_exe_polybius">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Open...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_run_polybius">
        <item>
         <spacer name="spacer_polybius_L">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_run_polybius">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Restart Process</string>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_polybius_R">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_irene">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="console_irene">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::WidgetWidth</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>   __ ____   ____ __  __  ____
   || || \\ ||    ||\ || ||
   || ||_// ||==  ||\\|| ||==
   || || \\ ||___ || \|| ||___

</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_exe_irene">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_exe_irene">
          <property name="text">
           <string>Executable Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_exe_irene">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_exe_irene"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_exe_irene">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Open...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_run_irene">
        <item>
         <spacer name="spacer_irene_L">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_run_irene">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Restart Process</string>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_irene_R">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
